test/Sentinel1Test.jl

```diff
@@ -0,0 +1,32 @@
+
+include("../src/separateLater/Sentinel1/Sentinel1.jl")
+import .Sentinel1
+
+@testset "Sentinel1.jl" begin
+     
+    ####### Generate test data ##############
+    createMetaData() = Sentinel1.MetaDataSLC("VV",1,Sentinel1.DateTime(2013,7,1,12,30,59),5405)
+    
+    function createSwathSLC() 
+        metaData = createMetaData() 
+        pixels = ones( Complex{Float64}, (2, 3)) .* (1.0+2.0im)
+        return Sentinel1.SwathSLC(metaData, (20,50),pixels)
+    end
+
+     ####### Test functions ##############
+    function constructMetaDataTest() 
+        testMeta = createMetaData()
+        return testMeta.polarisation == "VV" && testMeta.frequencyInMHz == 5405
+    end
+
+    function constructSwathSLCTest() 
+        testSwath = createSwathSLC()
+        return testSwath.metadata.swath == 1 && size(testSwath.pixels) == (2,3)
```

Comment:
Are we only using swath 1 to begin with, or is it just an example of a test? (similar question wrt. the other choices made here)

Reply:
We do all swaths. 

This is just a simple unit test. The purpose is just to check that there are no major errors in the struct and that the include/import statements are correct so that the code can run. 

Creating the same test with a different swath should activate the exact same part of the code so it would be a "duplicate"

Reply:
What about polarization mode in general metadata? Or that is not needed for now?

Reply:
It can be included here as `testSwath.metadata.polarisation` but this is just a simple test here.

The data struct in this PR is not complete. I am sure we will have to add fields along the way. I just pushed it as a beging.